### ISSUE TITLE
The loss does not decrease during training a flow model

### ISSUE THREAD

**Issue by yao5461** [yjxiong/temporal-segment-networks#8]
Hi,
I tried to train a optical flow model, follow this command:
```shell
bash scripts/train_tsn.sh ucf101 flow
```
But the loss does not decrease during training TSN. Any suggestions? By the way the rgb model is OK!
```
I0905 10:36:53.966650 24339 solver.cpp:240] Iteration 20, loss = 4.61511
I0905 10:36:53.966797 24339 solver.cpp:255]     Train net output #0: loss = 4.615 (* 1 = 4.615 loss)
I0905 10:36:53.966809 24339 solver.cpp:631] Iteration 20, lr = 0.005
I0905 10:37:39.902165 24339 solver.cpp:240] Iteration 40, loss = 4.61494
I0905 10:37:39.902279 24339 solver.cpp:255]     Train net output #0: loss = 4.61514 (* 1 = 4.61514 loss)
I0905 10:37:39.902290 24339 solver.cpp:631] Iteration 40, lr = 0.005
I0905 10:38:25.729789 24339 solver.cpp:240] Iteration 60, loss = 4.61493
I0905 10:38:25.729914 24339 solver.cpp:255]     Train net output #0: loss = 4.6159 (* 1 = 4.6159 loss)
I0905 10:38:25.729926 24339 solver.cpp:631] Iteration 60, lr = 0.005
```
```
I0905 22:13:58.890894 24339 solver.cpp:240] Iteration 19820, loss = 4.61208
I0905 22:13:58.891036 24339 solver.cpp:255]     Train net output #0: loss = 4.62788 (* 1 = 4.62788 loss)
I0905 22:13:58.891048 24339 solver.cpp:631] Iteration 19820, lr = 5e-05
I0905 22:14:39.989903 24339 solver.cpp:240] Iteration 19840, loss = 4.60292
I0905 22:14:39.990012 24339 solver.cpp:255]     Train net output #0: loss = 4.65203 (* 1 = 4.65203 loss)
I0905 22:14:39.990025 24339 solver.cpp:631] Iteration 19840, lr = 5e-05
I0905 22:15:21.071648 24339 solver.cpp:240] Iteration 19860, loss = 4.59892
I0905 22:15:21.071779 24339 solver.cpp:255]     Train net output #0: loss = 4.63011 (* 1 = 4.63011 loss)
I0905 22:15:21.071791 24339 solver.cpp:631] Iteration 19860, lr = 5e-05
```
Should I modify the solver file (tsn_bn_inception_flow_solver.prototxt)? change the learning rate?
Thanks!

**Comment by yjxiong** [comment]
Please see #7 also

**Comment by yao5461** [comment]
Thanks!
